test/unit/vl-modal.test.html

```diff
@@ -69,11 +72,32 @@
         assert(window.vl.modal.dress.calledWith(modal._dialogElement));
       });
 
-      test('de open methode called vl.modal.toggle met de dialog', () => {
+      test('de open methode called vl.modal.toggle met de dialog', (done) => {
         const modal = getModal(fixture('vl-modal-fixture'));
         modal.open();
-        assert(window.vl.modal.toggle.calledWith(modal._dialogElement));
-        assert(window.vl.modal.lastClickedToggle === modal._dialogElement);
+        setTimeout(() => {
```

Comment:
We kunnen hier dan ook gewoon awaitUntillen tot isConnected, nee?
Aan de andere kant, als we zeker zijn dat wachten tot de volgende event cycle voldoende is, is setTimeout ook niet fout...

Reply:
Het is pas in de setTimeout dat ik de stub op connected zet, ik wou namelijk eerst zeker zijn dat de call nog niet gebeurd was, vandaar deze setup.